wp-content/themes/core/pcss/content/panel/_gallery.pcss

```diff
@@ -6,38 +6,110 @@
 
 .site-panel--gallery {
 
-	.c-slider__slide {
-		aspect-ratio: '16:10';
+	.g-col {
+		padding-bottom: var(--spacer-2);
+
+		@media (--viewport-medium) {
+			padding-bottom: var(--spacer-4);
+		}
+	}
+}
+
+.site-panel--gallery__content {
+	max-width: 848px;
```

Comment:
Same notes on these max-widths or do you feel that these are so specific that they aren't worth moving into var form?

Reply:
These are very specific to the panels that I've noticed so far. If we do do this, I'd recommend some consistency from design if possible. I'm seeing this mostly within the panel / section header areas with the panel title and description.

Reply:
That's fair, could you follow up them and see, it's not a bad idea to have a consistent set in place ➕